NGitLab/IGitLabClient.cs

```diff
@@ -79,6 +79,11 @@ public interface IGitLabClient
 
     IMergeRequestClient GetMergeRequest(ProjectId projectId);
 
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    IMergeRequestClient GetGroupMergeRequest(int groupId);
```

Comment:
Why have the `[EditorBrowsable(EditorBrowsableState.Never)]` attribute here?

If we should avoid calling this method, then it simply should not be added to the interface. The only reason we have the attribute on, say, the `IMilestoneClient GetMilestone(int projectId)` declaration is because it was already part of the public API when we decided to make it "obsolete"...